internal/datanode/data_node.go

```diff
@@ -457,9 +457,9 @@ func (node *DataNode) Start() error {
 		},
 		Count: 1,
 	})
-	if err != nil {
-		log.Warn("fail to alloc timestamp", zap.Error(err))
-		return err
+	if rep.Status.ErrorCode != commonpb.ErrorCode_Success || err != nil {
```

Comment:
test err != nil first.
rep may be nil and access Status casue program panic.

Reply:
Has updated.